Azure/sonic-buildimage
# Arista qx 32s

**- What I did**
Add mmu buffer support for Arista-7050-QX-32S
**- How I did it**

**- How to verify it**
Restart swss service and dump PORT_PG_CONFIG mem to see if the buffer setting is properly loaded
**- Description for the changelog**

The qos config has been moved out to base image, so the td2.32ports.qos.json won't take effect. Ying has prepared the qos.json for the base image, and will probably check in later. For consistency with the previous setup lines, I include it there.

**- A picture of a cute animal (not mandatory but encouraged)**
